tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php

```diff
@@ -255,14 +256,17 @@ public function testGetReceiversOfShareReturnedBySharedByMe(): void
         $shares = $this->ocis->getSharedByMe();
         $this->assertCount(6, $shares);
         for($i = 0; $i < 6; $i++) {
-            $this->assertThat(
-                $shares[$i]->getReceiver()->getDisplayName(),
-                $this->logicalOr(
-                    $this->equalTo("philosophy-haters"),
-                    $this->equalTo("Marie Curie"),
-                    $this->equalTo("Albert Einstein")
-                )
-            );
+            if($shares[$i] instanceof ShareCreated) {
```

Comment:
there should be only `ShareCreated` items here at this time, or?
The danger with this is, that if we would have a bug and return something else than `ShareCreated` there would be no assertions done.
To make the analyzer happy maybe it would be enough to check `assertContainsOnly`

Reply:
There is https://docs.phpunit.de/en/9.6/assertions.html#assertinstanceof

So you could not put this `if` statement, but instead just have an `assertInstanceOf` before `assertThat` - that way the test will fail early if `$shares[$i]` is not an instance of `ShareCreated`